boot-passthrough.sh

```diff
@@ -6,8 +6,9 @@ args=(
   -machine pc-q35-2.9
   -smp 4,cores=2
   -vga none
-  -device vfio-pci,host=01:00.0,bus=pcie.0,multifunction=on
-  -device vfio-pci,host=01:00.1,bus=pcie.0
+  -device pcie-root-port,bus=pcie.0,multifunction=on,port=1,chassis=1,id=port.1
```

Comment:
@kholia this was found to be necessary on a Radeon RX4XX series card, is it possible this could break something? I couldn't get audio to work without this fix.

Reply:
I will have to give it a try. I don't have an ATI GPU atm.